drivers/usbc/tcpc/ucpd_stm32.c

```diff
@@ -1009,46 +1009,47 @@ static void ucpd_isr(const struct device *dev_inst[])
 	const struct tcpc_config *config;
 	struct tcpc_data *data;
 	uint32_t sr;
-	uint32_t ucpd_base;
 	struct alert_info *info;
 	uint32_t tx_done_mask = UCPD_SR_TXMSGSENT |
 				UCPD_SR_TXMSGABT |
 				UCPD_SR_TXMSGDISC |
 				UCPD_SR_HRSTSENT |
 				UCPD_SR_HRSTDISC;
 
-	if (IS_ENABLED(CONFIG_SOC_SERIES_STM32G0X)) {
+#ifdef CONFIG_SOC_SERIES_STM32G0X
```

Comment:
Nit: this doesn't match commit description

Reply:
The comment has been updated.